src/main/java/seedu/address/logic/commands/meetings/FindMeetingCommand.java

```diff
@@ -90,5 +91,16 @@ public CommandResult execute(Model model) throws CommandException {
         return personPred;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
It's better to split this line into two lines for coding standard. Already fix for you.